ci-operator/config/maistra/prometheus/maistra-prometheus-maistra-2.1.yaml

```diff
@@ -0,0 +1,25 @@
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.1"
+  use_build_cache: true
+resources:
+  '*':
+    limits:
+      memory: 8Gi
```

Comment:
remove this memory limit. The default cluster node may not has 8Gi memory.

Reply:
It worked previously. Other jobs have 8 and some even 16. Note that this is the limit, not the request.

Reply:
hm, I don't know other difference. 
You may fetch and merge recent master branch changes. And then run `make update` again. 
I guess they have changed some infrastructure settings and old master branch code  `make update` target need an update.